### ISSUE TITLE
Home -> Navigation -> Problems: “Add New …” Button Incorrectly Included in List 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#763]
When editing a problem set, the “Add new Problem” button is included as the last item of the Problems list.  This is semantically incorrect, as the button doesn’t represent a problem, but rather a mechanism with which to add content. 
 
Suggested Solution 
 
Remove the “Add new Problem” button from the list, rendering it before or after the list mark-up instead.

**Comment by jscholes** [comment]
@rupeshparab - the button has been moved out of the list, but looking at the code it seems to still be included in the sortable items.  Regardless of the fact that  sorting is currently not accessible, I don't think that the button to add a new problem should be treated in the same way as the problems themselves.  Semantically it is completely different.  I can see a case for allowing users to drag it before or after the list, but not within it.

In any case, the fact that it is being treated the same is causing a mark-up problem.  Namely, the annotation for the "Add New" button is also set to "Add New", meaning that there is a `<span>` with the same text before the button.

**Comment by rupeshparab** [comment]
@jscholes 
I checked it, the 'Add new' button was inside the SortableContainer but as it was not component of the SortableElement class it wasn't being sorted. But to make it clear in the code, I have moved it outside the SortableContainer.
Also, regarding the second 'Add New' text, it is due to a button with the 'Add New' container which is doing the same task as the container.
I am moving this ticket to QA, I have proposed the removal of nested button in 'Add new' container on #761, we can track its status there.

**Comment by jscholes** [comment]
@rupeshparab - sounds good to me.  Fixed in https://github.com/benetech/MathShare/commit/81895106ccad063f958c0e630be0ff11a562e303.
